### PR TITLE
Documentation 10 done. Doc 1 updated

### PR DESCRIPTION
**Pull request by sofigbarbes**

### CODE REVIEW

File: src/docs/10_quality_scenarios.adoc [hunk 1]
```diff
@@ -61,3 +63,23 @@ more precisely down to a level of scenarios that can be discussed and evaluated.
...
+[options="header",cols="1,2,1"]
+|===
+|Quality goal|Scenario|Priority
+|*_Usability_*  | Non-technical people should easily use the application. In case they want to look for information, the content should be found quickly. | High, medium
+|*_Confidentiality_*  | The control of user data should be decentralized. This way the user has a complete control of the data they share and receive. | High, High
+|*_Interoperability_*  | Our system should be able to exchange data with other systems and present that data so it can be understood by the user. | High, High
+|*_Integrity_*  | The application should't have vulnerable points: a third part shouldn't be able to access the data. | High, Medium
+|*_Performance efficiency_* | The response and processing times of the application should be quick. | High, Medium
+|*_Testability_* | The architecture should be designed in such a way that it succesfully passes the tests for its implementation. | High, Medium
+|*_Modifyability_* | As Solid is a pioneer evolutive technology, our application should be able to accept changes and be modifyed. | Medium, Low
```
Comment (blluci) [inline src/docs/10_quality_scenarios.adoc]: Modifiability* and modified*

File: src/docs/10_quality_scenarios.adoc [hunk 2]
```diff
@@ -36,6 +36,8 @@ The quality tree is a high-level overview of the quality goals and requirements:
 In any case the tree should include links to the scenarios of the following section.
 ****
 
+= image:qualityTree.jpg[Quality tree]
```
Comment (blluci) [inline src/docs/10_quality_scenarios.adoc]: I think to add images in ascii doc you should use "image:: "

File: src/docs/10_quality_scenarios.adoc [hunk 3]
```diff
@@ -61,3 +63,23 @@ more precisely down to a level of scenarios that can be discussed and evaluated.
...
+* _Scenario_: Describe how the system should behave. It specifies quality requirements.
+* _Priority_: It has two values: importance for success, and how difficult it is to be achieved. 
+
+[options="header",cols="1,2,1"]
+|===
+|Quality goal|Scenario|Priority
+|*_Usability_*  | Non-technical people should easily use the application. In case they want to look for information, the content should be found quickly. | High, medium
+|*_Confidentiality_*  | The control of user data should be decentralized. This way the user has a complete control of the data they share and receive. | High, High
+|*_Interoperability_*  | Our system should be able to exchange data with other systems and present that data so it can be understood by the user. | High, High
+|*_Integrity_*  | The application should't have vulnerable points: a third part shouldn't be able to access the data. | High, Medium
```
Comment (blluci) [inline src/docs/10_quality_scenarios.adoc]: Shouldn't* , a third **party***

File: src/docs/10_quality_scenarios.adoc [hunk 4]
```diff
@@ -61,3 +63,23 @@ more precisely down to a level of scenarios that can be discussed and evaluated.
...
+A table with quality goals and concrete scenarios, ordered by priorities.
+
+* _Quality goal_: Annotate and qualify the functionality.
+* _Scenario_: Describe how the system should behave. It specifies quality requirements.
+* _Priority_: It has two values: importance for success, and how difficult it is to be achieved. 
+
+[options="header",cols="1,2,1"]
+|===
+|Quality goal|Scenario|Priority
+|*_Usability_*  | Non-technical people should easily use the application. In case they want to look for information, the content should be found quickly. | High, medium
```
Comment (blluci) [inline src/docs/10_quality_scenarios.adoc]: Medium should start with uppercase to maintain consistency

File: src/docs/10_quality_scenarios.adoc [hunk 5]
```diff
@@ -61,3 +63,23 @@ more precisely down to a level of scenarios that can be discussed and evaluated.
...
+|*_Confidentiality_*  | The control of user data should be decentralized. This way the user has a complete control of the data they share and receive. | High, High
+|*_Interoperability_*  | Our system should be able to exchange data with other systems and present that data so it can be understood by the user. | High, High
+|*_Integrity_*  | The application should't have vulnerable points: a third part shouldn't be able to access the data. | High, Medium
+|*_Performance efficiency_* | The response and processing times of the application should be quick. | High, Medium
+|*_Testability_* | The architecture should be designed in such a way that it succesfully passes the tests for its implementation. | High, Medium
```
Comment (blluci) [inline src/docs/10_quality_scenarios.adoc]: Successfully*

File: src/docs/01_introduction_and_goals.adoc [hunk 6]
```diff
@@ -62,19 +62,12 @@ A table with quality goals and concrete scenarios, ordered by priorities
...
-|*_Performance efficiency_* | The response and processing times of the application should be quick. | High, Medium
-|*_Maintainability_* | As Solid is a pioneer evolutive technology, our application should be able to accept changes and be maintained. | Medium, Low
-|*_Availability_* | Online independence: The user should be able to use the application without need of Internet connection. | Medium, Medium
-|*_Portability_* | The system could work on a mobile device. | Low, High
-
-
-
+|Quality goal|Motivation
+|*_Confidentiality_*  | The control of user data should be decentralized. 
+|*_Security_* | The application should't have vulnerable points.
```
Comment (blluci) [inline src/docs/01_introduction_and_goals.adoc]: Shouldn't*

File: src/docs/10_quality_scenarios.adoc (hunk 2 above)
Comment (sofigbarbes) [reply src/docs/10_quality_scenarios.adoc]: It was:
=image:images/image.jpg[altText]
Thanks!

**Pull request merged**